<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>62F6C9E2BF8D128C43E905262881F6801C50178A_Html</title><meta name="author" content="x c"/><style type="text/css"> * {margin:0; padding:0; text-indent:0; }
 h1 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 24pt; }
 .s1 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 14pt; }
 .s2 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 9pt; vertical-align: 5pt; }
 h2 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 18pt; }
 p { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; margin:0pt; }
 .s4 { color: black; font-family:SimSun; font-style: normal; font-weight: normal; text-decoration: none; font-size: 14pt; }
 h3 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 14pt; }
</style></head><body><h1 style="padding-top: 2pt;padding-left: 14pt;text-indent: 0pt;line-height: 112%;text-align: center;">Genome-wide DNA Methylation Analysis Identifies Epigenetic Markers and Barriers of Nuclear Reprogramming</h1><p class="s1" style="padding-top: 10pt;padding-left: 6pt;text-indent: 0pt;line-height: 191%;text-align: justify;">Mingmin Song<span class="s2">1</span>, Yu Feng<span class="s2">1</span>, Pengbo Cao<span class="s2">1</span>, Lei Zheng<span class="s2">1</span>, <b>Xing Chen</b><span class="s2">1</span>, Guangpeng Li<span class="s2">1</span>, Lei Yang<span class="s2">1</span>, Yongchun Zuo<span class="s2">1,</span></p><p class="s2" style="padding-top: 5pt;padding-left: 6pt;text-indent: 0pt;line-height: 194%;text-align: justify;">1 <span class="s1">State key Laboratory of Reproductive Regulation and Breeding of Grassland Livestock, College of Life Sciences, Inner Mongolia University, Hohhot, 010070, China.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h2 style="padding-top: 8pt;padding-left: 6pt;text-indent: 0pt;text-align: left;">Abstract:</h2><p style="padding-top: 10pt;padding-left: 6pt;text-indent: 27pt;line-height: 150%;text-align: justify;">Mammalian oocytes can reprogram somatic cells into a totipotent state enabling animal cloning through somatic cell nuclear transfer (SCNT). However, the majority of SCNT embryos will developmental arrest in the early stages due to many defects in reprogramming process. For the problem of low rate of nuclear transfer embryos, the focus of research is basically on the molecular mechanism of early embryonic development of SCNT. To date, only the reduction of H3K9me3 by ectopic expression of histone demethylases has been shown to greatly promote the development of SCNT embryos, and its potential molecular regulation mechanism has not yet been precisely resolved.</p><p style="padding-left: 6pt;text-indent: 27pt;line-height: 150%;text-align: justify;">In this paper, we first comprehensively analyzed the transcriptome expression profiles of normal fertilized and somatic cell nuclear transfer embryos in mice, with systematic analysis of molecular differences in whole gene expression in different developing fate cloned embryos, We found that abnormal activation of important functional pathways such as apoptosis, autophagy, endocytosis, and DNA repair is a key factor for 2-cell arrest of nuclear transfer embryos; and abnormal activation of stem cell population maintence, DNA repair, cell cycle, and autophagy is a molecular marker for 4-cell arrest of cloned embryos. By comparing the transcriptomes between SCNT embryos with different fate, some nuclear reprogramming related gene clusters of 2-cell and 4-cell embryos were screened out from the angles of transcription factors, pluripotency maintenance gene, histone apparent modification factor, DNA/RNA demethylase gene etc: Ascl2<span class="s4">、</span>Bmp7<span class="s4">、</span>Kdm4d<span class="s4">、</span>Kat2b<span class="s4">、</span>Srcap and Zfp217 belong to the key regulatory factors of cloned embryo 2-cell; Sall4<span class="s4">、</span>Vegfa<span class="s4">、</span> Hinfp<span class="s4">、</span>Kdm3b<span class="s4">、</span>Kdm5a<span class="s4">、</span>Kdm5b<span class="s4">、</span>Ctnnb1<span class="s4">、</span>Hdac4<span class="s4">、</span>Brca2<span class="s4">、</span>Yeats4<span class="s4">、 </span>Atxu7 and Arid4b play an important regulatory  role in the 4-cell of cloned embryo; Zscan4c<span class="s4">、</span>Id1<span class="s4">、</span>Pou6f1<span class="s4">、</span>Cited1<span class="s4">、</span>Nanog<span class="s4">、</span>Dppa2<span class="s4">、</span>Fgf4<span class="s4">、</span>Med17<span class="s4">、</span>Cnot1<span class="s4">、</span>Kdm4b<span class="s4">、</span>Kdm3a<span class="s4">、</span>Kdm7a<span class="s4">、</span>Kat8<span class="s4">、</span>Elp6<span class="s4">、</span>Eid1<span class="s4">、</span>Gadd45a<span class="s4">、 </span>Tet1 and Tet2 belong to the molecular markers that the cloned embryos are not normally activated due to the differences in 2-cell and 4-cell between normal embryos and different cloned embryos. and a small number of genes have been reported. These molecular differences may explain the phenomenon of arrest of SCNT embryos.</p><p style="padding-left: 6pt;text-indent: 27pt;line-height: 150%;text-align: justify;">In addition, we compared and analyzed the gene expression differences of histone demethylases Kdm4b/4d and Kdm5b in overcoming the embryonic developmental arrest of nuclear transfer and promoting cloned embryonic developmental potential. and found that activation of key pathways and core gene clusters played an important role. Kdm4b/4d can promote the expression of the key regulatory factors Zscan4c and Zscan4d that maintain telomeres in 2-cells. In addition, we also found that Kdm4d can specifically activate Kdm5b <span class="s4">、</span>Tet1 and Id1 in the 2-cell stage, which indicates that the molecular markers of Kdm4d significantly improves the blastocyst rate of SCNT  embryo cells are identificated. Moreover, we discovered that kdm5b could activate the pathways of stem cell maintenance and DNA repair and expression of key genes of Ubtfl1<span class="s4">、</span>Nr5a2<span class="s4">、</span>Prdm14<span class="s4">、</span>Thoc5<span class="s4">、</span>Dppa3 and Klf5 in 4-cell, the activation of these key genes may be one of the important factors that histone demethylase can overcome the blockage of SCNT embryos.</p><p style="padding-left: 6pt;text-indent: 41pt;line-height: 150%;text-align: justify;">Our findings provide important theoretical basis for exploring the potential molecular mechanisms and key factors in SCNT-mediated somatic cell reprogramming, and provide some help for further elucidating the molecular regulation mechanisms of cell reprogramming.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h3 style="padding-left: 6pt;text-indent: 0pt;line-height: 150%;text-align: justify;">Keywords: Somatic cell nuclear transfer; Embryonic development arrest; Transcriptome different express analysis; Histone demethylases; Key molecular markers</h3><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 20pt;text-indent: 0pt;text-align: left;"><span><img width="518" height="280" alt="image" src="./Image_003.jpg"/></span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 187pt;text-indent: -167pt;line-height: 194%;text-align: left;">A simple model for key activation pathways and genes in NT to blast embryos.</p></body></html>
