<div class="golf">
	<div class="photo"><img src="./paper_222.jpg"></div>
	<div class="intro">
	    <h3>Study of Binding Profiles of Key Transcriptional Factors in LncRNA Region during Mouse Somatic Cell Reprogramming</h3>
	    <p>    Induced pluripotent stem cells (iPSCs) have a promising prospect of implications for regenerative medicine, disease modeling and drug screening. Nevertheless, the inefficiency of reprogramming blocks the implications of iPSC. Key transcriptional factors (TFs) in somatic cell reprogramming like Oct4, Sox2, Klf4, cMyc (OSKM) play a critical and pioneering regulartory role in somatic cell identity transformation. Previous studies on the targeted regulation of OSKM in somatic cell genome reprogramming mainly focused on the promoter regulatory region of protein-coding genes and screened out some substitutable key genes. Recently, an increasing number of evidence indicate that long non-coding RNAs (lncRNAs) play an equal important regulatory role in somatic cell reprogramming, yet the dynamic targeted regulation patterns of key TFs in this region have not been systematically analyzed and studied.
	    <p>    Here, at first we mapped chromosome distribution patterns of targeted binding of six TFs Oct4, Sox2, Klf4, cMyc, Nanog and Esrrb at key timepoints of mouse somatic cell reprogramming, showing that the regulation of pluripotent TFs was mainly on autosome 9, 10, 13 and 17, and tended to bind to lincRNA with promoter containing endogenous retrovirus K (ERVK). Further, combining with ATAC-seq data, we found that OSKM successively co-bind to genome-wide target sites in time order during somatic cell reprogramming, and occupancy of OSKM was consistent with opening of chromatin. Next, combining with TF-binding motif analysis, we found that binding of somatic cell specific TFs on lncRNA promoter and enhancer regions declined during reprogramming, but that of pluripotency TFs gradually increased; OSKM binding on promoter cooperated with histone modifications H3K4me3, H3K27me3, H3K79me2 and histone variant H3.3 to regulate the transcription of lncRNA; In addition, we explored the relationship between preference of OSKM binding on promoter and lncRNA characteristics, including lncRNA type, transposable element (TE) content, transcript length, first exon length, first intron length, and distance to pluripotency genes. It was found that OSKM preferentially bind to lncRNAs of types those overlapped with protein coding sequences, such as sense overlapping and antisense, and lncRNAs containing ERVK in promoter, longer transcript, longer first intron, and within 10-20 kb range of pluripotency genes.
            <p>    Further, based on correlation analysis of iPSC pluripotency protein-coding (PC) genes and lncRNA, 210 iPSC pluripotency network related lncRNAs were found, a PC-lncRNA genes co-expression network was constructed, and several key lncRNAs probably play vital roles in the network were obtained. Among them, Platr family, Panct2, D230017M19Rik and 4930461G14Rik have been confirmed, which to some extent confirms the reliability of our results. Meanwhile, we also excavated a series of new lncRNAs, such as Tnfsf13os, Gm44699, Gm26917, Gm31793, Gm16302, Gm48604, etc. Together, we depicted binding profiles of key TFs in lncRNA region during mouse somatic cell reprogramming, studied the relationship of co-regulation of lncRNA transcription between key TFs and epigenetic modifications, and built a iPSC pluripotency related PC-lncRNA genes co-expresssion network. Our findings may be significant in illuminating molecular mechanisms of key TFs regulating lncRNA, and may provide some theoretic help for further identification of non-coding RNA reprogramming molecular markers.
	 
                        </p>
	</div>
</div>
<style>
	.photo{
		float:left;
		width:40%;
	}
<style>
	
<style>
	.intro{
		float:right;
		width:50%;
	}
<style>

	
