
<!DOCTYPE  html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><title>C2FE57C79522B24FAAC85459028C19CF381F987B_Html</title><meta name="author" content="x c"/><style type="text/css"> * {margin:0; padding:0; text-indent:0; }
 h1 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 20pt; }
 .s1 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 12pt; }
 .s2 { color: black; font-family:"Times New Roman", serif; font-style: italic; font-weight: normal; text-decoration: none; font-size: 8pt; vertical-align: 4pt; }
 h2 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 16pt; }
 p { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; margin:0pt; }
 .a { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: normal; text-decoration: none; font-size: 12pt; }
 h3 { color: black; font-family:"Times New Roman", serif; font-style: normal; font-weight: bold; text-decoration: none; font-size: 12pt; }
</style></head><body><h1 style="padding-top: 10pt;padding-left: 26pt;text-indent: 0pt;line-height: 150%;text-align: center;">Machine Learning of Single-cell Transcriptome Highly Identify Development Signature by Comparing F-score Selection with Differential Expression Analysis</h1><p class="s1" style="padding-top: 8pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Pengfei Liang<span class="s2">1</span>, <b>Xing Chen</b><span class="s2">1</span>, Chunshen Long<span class="s2">1</span>, Lei Zheng<span class="s2">1</span>, Lei Yang<span class="s2">2</span>,* and Yongchun Zuo<span class="s2">1</span>,*</p><p style="text-indent: 0pt;text-align: left;"><br/></p><p class="s2" style="padding-left: 5pt;text-indent: 0pt;line-height: 170%;text-align: left;">1 <span class="s1">State key Laboratory of Reproductive Regulation and Breeding of Grassland Livestock, College of Life Sciences, Inner Mongolia University, Hohhot, 010070, China.</span></p><p class="s2" style="padding-top: 4pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">2 <span class="s1">College of Bioinformatics Science and Technology, Harbin Medical University, Harbin 150081, China.</span></p><p class="s2" style="padding-top: 12pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">* <span class="s1">Corresponding author.</span></p><p style="text-indent: 0pt;text-align: left;"><br/></p><h2 style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Abstract:</h2><p style="padding-top: 9pt;padding-left: 5pt;text-indent: 24pt;line-height: 150%;text-align: justify;"><a href="http://bioinfor.imu.edu.cn/empredictor/" class="a" target="_blank">Human preimplantation development is a complex process involving dramatic changes in the transcriptional architecture. Single-cell transcriptome analysis plays important roles in signature identification of transcription atlas in embryo development. However, the traditional analysis pipelines of enormous transcriptome datasets are not fully effective on single-cell RNA-Seq techniques. In this study, we firstly compared F-score of feature selection algorithms with differential gene expression (DGE) analysis to find specific signatures of development stage. By using machine learning evaluation, four types of signature subset were comprehensively discussed. The prediction results showed that a feature subset with 1881 genes based on F-score algorithm obtained that best predictive performance, which achieved the highest accuracy of 0.93 on the cross validation set. Further function enrichment demonstrated that the gene set selected by the feature selection method involved in more development-related pathways and cell fate determination biomarkers. It indicates that F-score algorithm should be preferential proposed for detecting key genes of multi-period data. At last, a free online server was established for the convenience of most experimental scientists at </a>http://bioinfor.imu.edu.cn/empredictor/. A friendly guide is provided to describe how to use the web server.</p><p style="text-indent: 0pt;text-align: left;"><br/></p><h3 style="padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: left;">Keywords: Embryo development, Expression signature, Single-cell transcriptome, Machine learning, Feature selection, Identification, Web server</h3><p style="text-indent: 0pt;text-align: left;"><br/></p><p style="padding-left: 5pt;text-indent: 0pt;text-align: left;"><span><img width="559" height="429" alt="image" src="C2FE57C79522B24FAAC85459028C19CF381F987B_Html/Image_001.jpg"/></span></p><p style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;line-height: 112%;text-align: justify;">The workflow consists of four steps<b>. </b>Firstly, using different feature selection methods found DEGs. Secondly, DEGs was trained by support vector machine. Thirdly, the model performances were compared to each other to evaluate different feature selection methods. Last, we use the best results to obtain biomarker of different stages.</p></body></html>
